spec/features/user_show_spec.rb

```diff
@@ -101,4 +101,4 @@
       expect(current_path).to eq(user_posts_path(@su))
     end
   end
-end
\ No newline at end of file
+encouraged
```

Comment:
- [ ] This line here prevents the code from running correctly as can be seen in the attached image below 👇 
Kindly remove `encouraged` and replace it with `end` 👍 

![image](https://github.com/sheezu900/blog-rails/assets/46839548/3bd552ca-6a5e-425b-abdb-800fae3f58cd)